infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/ValidatorLogger.java

```diff
@@ -170,7 +170,7 @@ private String formatBlockRoots(final Set<Bytes32> blockRoots) {
     return blockRoots.stream().map(LogFormatter::formatHashRoot).collect(Collectors.joining(", "));
   }
 
-  public void aggregationSkipped(final UInt64 slot, final int committeeIndex) {
+  public void aggregationSkipped(final UInt64 slot, final UInt64 committeeIndex) {
```

Comment:
we only see this used in test code in this PR which is curious... if its only testing code we should probably move it, and Im less concerned with the interface change, otherwise i think committeeIndex would have been better to stay as int :)

Reply:
weird that interface changes and calling point doesnt...